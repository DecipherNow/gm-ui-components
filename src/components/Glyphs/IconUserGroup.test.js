import React from "react";
import { shallow } from "enzyme";

import IconUserGroup from "./IconUserGroup";

describe("IconUserGroup", () => {
  it("matches snapshot", () => {
    const aIconUserGroup = shallow(<IconUserGroup />);
    expect(aIconUserGroup).toMatchSnapshot();
  });
});