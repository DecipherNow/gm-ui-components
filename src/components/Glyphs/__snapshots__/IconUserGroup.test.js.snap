// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconUserGroup matches snapshot 1`] = `<IconMock />`;
